packages/components/organisms/f-header/test/visual/f-header.visual.desktop.spec.js

```diff
@@ -0,0 +1,97 @@
+import forEach from 'mocha-each';
+
+const { buildUrl } = require('@justeat/f-wdio-utils/src/storybook-extensions.js');
+
+const Header = require('../../test-utils/component-objects/f-header.component');
+
+let header;
+
+describe('Shared - f-header component tests', () => {
+
+    forEach([['en-GB', true], ['en-GB', false],
+        ['en-AU', true], ['en-AU', false],
+        ['en-NZ', true], ['en-NZ', false],
+        ['en-IE', true], ['en-IE', false],
+        ['it-IT', true], ['it-IT', false],
+        ['es-ES', true], ['es-ES', false]
+    ])
+    .it('should display component', (tenant, isLoggedIn) => {
+        // Arrange
+        header = new Header('organism', 'header-component');
+        header.withQuery('&knob-Locale', tenant);
+        // Both props below should only show for UK
+        header.withQuery('&knob-Show offers link', 'true');
+        header.withQuery('&knob-Show delivery enquiry', 'true');
+
+        if (!isLoggedIn) {
+            header.withQuery('&knob-User info', isLoggedIn);
+        }
+        const pageUrl = buildUrl(header.componentType, header.componentName, header.path);
```

Comment:
Not part of your PR but if we are always passing in these three parameters via properties of the components couldn't we simply pass the component object and restructure the values out in the `buildUrl` function? i.e. 

```
const buildUrl = ({ componentType, componentName, path }) => {
```

Seems like it would save a lot of typing in these tests.